client/src/pages/EditProfilePage/EditProfilePage.tsx

```diff
@@ -21,32 +22,167 @@ const EditProfilePage = () => {
   const fileInputRef = useRef<HTMLInputElement>(null);
 
   const [formData, setFormData] = useState({
```

Comment:
It looks like this local state is very similar to the userProfile global state. 

I see you are using formData to eventually construct the payload for an action. I see a few ways you can approach this functionality without the need of local state.
- You'll need to refactor or create a few reducers
- Save updated profile to DB when user clicks submit
- Handle case for when users edit a couple of fields but decides to cancel / exit the page

Reply:
@JimmyTran48 I like this idea - I would suggest we move this to a subsequent ticket along with several other redux migration tickets.

Reply:
@JimmyTran48 Good suggestion. I discussed this with @brok3turtl3 and we are gonna go ahead and make another ticket for this.